VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.html

```diff
@@ -1,136 +1,210 @@
+      *ngIf="!screenShareStream && canShowScreenShareButton"
+      appTooltip
+      [text]="'private-consultation-room-controls.share-my-screen' | translate"
+      colour="grey"
+      (click)="startScreenShare()"
+    >
+      <span class="fak fa-start-screen-share"></span>
+    </div>
+
+    <div
+      *ngIf="screenShareStream"
+      class="icon-button"
+      id="stop-screenshare-img"
+      appTooltip
+      [text]="'private-consultation-room-controls.stop-sharing-my-screen' | translate"
+      colour="red"
+      (click)="stopScreenShare()"
+    >
+      <span class="fak fa-stop-screen-share"></span>
+    </div>
+
+    <div class="flexBar"></div>
+
+    <div
+      class="icon-button"
+      id="toggle-self-view-img"
+      appTooltip
+      *ngIf="!audioOnly"
+      [text]="
+        (selfViewOpen
+          ? 'private-consultation-room-controls.selfview-toggle-true'
+          : 'private-consultation-room-controls.selfview-toggle-false'
+        ) | translate
+      "
+      colour="grey"
+      (click)="toggleView()"
+    >
+      <span class="far" [class]="selfViewOpen ? 'fa-eye' : 'fa-eye-slash'"></span>
+    </div>
+
+    <div
+      class="icon-button"
+      id="toggle-video-mute-img"
+      appTooltip
+      *ngIf="!isJudge && !audioOnly"
+      [text]="
+        (videoMuted
+          ? 'private-consultation-room-controls.video-mute-toggle-true'
+          : 'private-consultation-room-controls.video-mute-toggle-false'
+        ) | translate
+      "
+      colour="grey"
+      (click)="toggleVideoMute()"
+    >
+      <span class="far" [class]="videoMuted ? 'fa-video-slash' : 'fa-video'"></span>
+    </div>
+
+    <div
+      class="icon-button"
+      id="toggle-audio-mute-img"
+      appTooltip
+      *ngIf="!remoteMuted || isJudge"
+      [text]="
+        (audioMuted
+          ? 'private-consultation-room-controls.audio-mute-toggle-true'
+          : 'private-consultation-room-controls.audio-mute-toggle-false'
+        ) | translate
+      "
+      colour="grey"
+      (click)="toggleMute()"
+    >
+      <span class="far" [class]="audioMuted ? 'fa-microphone-slash' : 'fa-microphone'"></span>
+    </div>
+
+    <div
+      class="icon-button"
+      id="toggle-audio-mute-locked-img"
+      appTooltip
+      *ngIf="remoteMuted && !isJudge"
+      [text]="'private-consultation-room-controls.remote-muted' | translate"
+      colour="grey"
+    >
+      <span class="far fa-microphone-slash red"></span>
+    </div>
+
+    <div
+      class="icon-button"
+      id="toggle-hand-raised-img"
+      appTooltip
+      *ngIf="!isPrivateConsultation && !isJudge"
+      [text]="
+        (handRaised
+          ? 'private-consultation-room-controls.hand-raise-toggle-true'
+          : 'private-consultation-room-controls.hand-raise-toggle-false'
+        ) | translate
+      "
+      colour="grey"
+      (click)="toggleHandRaised()"
+    >
+      <span class="far fa-hand-paper" [class]="{ yellow: handRaised }"></span>
```

Comment:
what fixed this?

Reply:
nothing, it's still the original implementation